src/core/server/integration_tests/config/check_dynamic_config.test.ts

```diff
@@ -16,128 +16,135 @@ import {
   request,
 } from '@kbn/core-test-helpers-kbn-server';
 import { PLUGIN_SYSTEM_ENABLE_ALL_PLUGINS_CONFIG_PATH } from '@kbn/core-plugins-server-internal/src/constants';
-
-describe('PUT /internal/core/_settings', () => {
-  let esServer: TestElasticsearchUtils;
-  let root: Root;
-
-  const loggerName = 'my-test-logger';
-
-  beforeAll(async () => {
-    const settings = {
-      coreApp: { allowDynamicConfigOverrides: true },
-      logging: {
-        loggers: [{ name: loggerName, level: 'error', appenders: ['console'] }],
-      },
-      server: { restrictInternalApis: false },
-    };
-    const { startES, startKibana } = createTestServers({
-      adjustTimeout: (t: number) => jest.setTimeout(t),
-      settings: {
-        kbn: settings,
-      },
+import { getFips } from 'crypto';
+
+if (getFips() === 0) {
```

Comment:
__question:__ Here and in all similar places: is this a temporary measure? If so, are we tracking all the tests that were disabled somewhere to ensure we remember to re-enable them at some point? If it’s not temporary, could we add a comment explaining why we’re doing this and what areas are affected?

Reply:
I cant say it is temporary; I had to disable oss so xpack plugins would register and allow `xpack.security....` configs to be allowed.

I think we would have to change the jest integration tests to allow me to add less broad-sweeping overrides. 

I am not keeping track per se, but the search to find them is pretty straightforward.

the test in the `else` (FIPS portion) has the explanation (below), but Im happy to add a comment if you think that is clearer:
`'is running in FIPS mode, skipping tests since they fail due to FIPS overrides'`

Reply:
I see, I was just wondering if there was a clear, describable reason why certain tests fail in FIPS, like, for example, “dynamic config relies on MD5 for … and is hence not compatible with FIPS”. But if that’s not the case, or if it’s too much work to pinpoint and describe the reason for every such exception, feel free to keep things as they are, it's not worth it.